engine/test/e2e/README.md

```diff
@@ -0,0 +1,30 @@
+## Preparations
```

Comment:
Better to use debug scripts
```bash
$ configs="./deployments/engine/docker-compose/3m3e.yaml ./deployments/engine/docker-compose/dm_databases.yaml"
$ engine/test/integration_tests/run.sh debug ${configs}

// Now we are in e2e dir
go test -v -race -run="TestDMJob"
```

Reply:
Ok. You can give me a suggestion. I will merge it.